K2Bridge/Controllers/IndexListController.cs

```diff
@@ -32,14 +32,14 @@ public IndexListController(IKustoDataAccess kustoDataAcess, ILogger<IndexListCon
         private ILogger Logger { get; set; }
 
         /// <summary>
-        /// Process the request.
+        /// Resolve the index.
         /// </summary>
-        /// <param name="indexName">The index to process.</param>
+        /// <param name="indexName">The index pattern to process.</param>
         /// <returns>The table list in the Kusto database.</returns>
         [Produces("application/json")]
-        public async Task<IActionResult> Process(string indexName)
+        public async Task<IActionResult> Resolve(string indexName)
```

Comment:
See my comment below, I think you can have the routing logic applied here as decoration.

Reply:
👍🏻